scripts/prepare_navit_macos.sh

```diff
@@ -0,0 +1,19 @@
+# Use this script to build navit on a Mac
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
```

Comment:
Don't auto install things which could be run by end-users, only do this when $CI is set to true

Reply:
My target was to create a solution that an end user can apply to get navit up and running without fidling around with git etc. which an enduser most probably don't know.

Reply:
For that we can build with your scripts on CI and provide the resulting binary on https://download.navit-project.org/

Reply:
Great. I was not aware of that. Maybe we can find a way to use my developer certificate to sign the build then (I could provide an ssh access to upload the app package to my machine, hava a service running checking for a new upload, sign it and somehow trigger CI to download the result again) and even bring navit to the app store. Don't know if any license restrictions will prohibit that. I guess there was some discussion in one of the issue conversations about that.